src/Types/JsonWsp.purs

```diff
@@ -153,6 +196,33 @@ parseMirror = caseJsonObject (Left (TypeMismatch "expected object")) $
       pure { step, id }
   )
 
+newtype Assets = Assets (Map CurrencySymbol (Map TokenName BigInt))
+
+instance DecodeJson Assets where
+  decodeJson j = do
+    assets :: Array (String /\ String) <- toUnfoldable <$> decodeJson j
```

Comment:
I only took a brief look, so this might be completely off, but:

I think the parse issue you're having might be coming from the `Map` decoder ([here](https://github.com/purescript-contrib/purescript-argonaut-codecs/blob/b0a041d92bfd548e2cd793cc7c02363464325a13/src/Data/Argonaut/Decode/Decoders.purs#L190)) which expects the JSON to be a list of "tuples"

Reply:
I thought so too, but then I directly tried `decodeMap`